pom.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>dev.samstevens.totp</groupId>
+  <groupId>com.github.moddx.totp</groupId>
```

Comment:
Maybe not change this? Seems that @samdjstevens isn't looking at this library anymore though, we'd like an update of dependencies too.